content/Assets/Styles/components/form/_checkbox.scss

```diff
@@ -0,0 +1,89 @@
+/* ==========================================================================
```

Comment:
This file should be named `checkboxes` to match the primary selector (comment should be updated to).